### PR TITLE
Close not called on the qml plugin

### PR DESCRIPTION
**Pull request by femdom**

### CODE REVIEW

File: snapcraft/plugins/qml.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def _build_qt_config(self):
         config = open(os.path.join(configdir, 'snappy-qt5.conf'), 'w')
         config.write('./usr/lib/{}/qt5/bin\n'.format(arch))
         config.write('./usr/lib/{}\n'.format(arch))
-        config.close
+        config.close()
```
Comment (kyrofa) [inline snapcraft/plugins/qml.py]: Huh, this must have sneaked by review. The fix should really be using a `with` though-- mind implementing that instead?

Comment (femdom) [inline snapcraft/plugins/qml.py]: Oh. Sorry. Updated a diff for that.

**Comment by kyrofa** [comment]
@femdom much better, thank you. :+1: from me once you're able to do the following (from the [contribution guide](https://github.com/ubuntu-core/snapcraft/blob/master/CONTRIBUTING.md))

1. Sign the contributor license agreement, making sure your commit email is included in the LP emails.
2. Squash.

Thank you again!

**Comment by kyrofa** [comment]
@femdom alright I've verified the CLA and you obviously squashed, thank you. We just need one more pair of eyes before this is merged.

**Comment by femdom** [comment]
@kyrofa , thanks!

**Comment by sergiusens** [comment]
:+1:

**Comment by sergiusens** [comment]
Care to cherrypick this for 1.x?

**Pull request merged**